src/get-start-and-end-commands.js

```diff
@@ -31,13 +32,23 @@ module.exports = async function getStartAndEndCommands({
     });
   }
 
-  let [
-    startCommand,
-    endCommand
-  ] = await Promise.all([
-    reset || init ? null : _prepareCommand(startOptions),
-    _prepareCommand(endOptions)
-  ]);
+  let startCommand;
+  let endCommand;
+
+  if (os.platform() === 'win32') {
+    // running two npx commands in parallel on Windows causes it to break the npx cache
+    // folders and throw strange permissions errors. We need to run them in series
+    startCommand = await reset || init ? null : _prepareCommand(startOptions);
```

Comment:
Can you put these in helper functions, so in both cases we just have `prepareStart()` and `prepareEnd()`?